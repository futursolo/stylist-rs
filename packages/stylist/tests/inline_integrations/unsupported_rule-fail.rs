fn main() {
    let is_an_expression = "black";
    let _ = stylist::css! {
        @page {
            margin: 1cm;
        }
        @property property-name {
            syntax: "<color>";
            inherits: false;
            initial-value: #c0ffee;
        }
        @completely-unknown {
            some-attribute: foo-value;
        }
    };
}
