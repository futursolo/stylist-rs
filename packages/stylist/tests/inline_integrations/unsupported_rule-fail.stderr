error: @-rule 'page' not supported
 --> $DIR/unsupported_rule-fail.rs:4:10
  |
4 |         @page {
  |          ^^^^

error: @-rule 'property' not supported
 --> $DIR/unsupported_rule-fail.rs:7:10
  |
7 |         @property property-name {
  |          ^^^^^^^^

error: @-rule 'completely-unknown' not supported
  --> $DIR/unsupported_rule-fail.rs:12:10
   |
12 |         @completely-unknown {
   |          ^^^^^^^^^^^^^^^^^^
